src/plugins/intel_cpu/tests/functional/behavior/ov_executable_network/properties.cpp

```diff
@@ -137,6 +137,24 @@ TEST_F(OVClassConfigTestCPU, smoke_CpuExecNetworkCheckModelStreamsHasHigherPrior
     ASSERT_EQ(streams, value);
 }
 
+#if !(defined(OPENVINO_ARCH_ARM) || \
```

Comment:
Proper approach is not to disable the test at all, but determine resulting value based on CPU arch. That will lead to failed test once ARM behavior is fixed, wo won't forget to upated the test.

Reply:
Updated